src/main/java/crazypants/enderio/machine/farm/FarmersRegistry.java

```diff
@@ -191,6 +192,16 @@ private static void addExtraUtilities() {
     }
   }
 
+  private static void addFlowers() {
+    FarmersCommune.joinCommune(new FlowerPicker().add(
+        GameRegistry.findBlock("minecraft", "yellow_flower"), 
+        GameRegistry.findBlock("minecraft", "red_flower"), 
```

Comment:
Oh, that.

Nicer pattern with the rest, easier to move to a config file in the future.